leaderboard/src/lib.rs

```diff
@@ -44,18 +44,26 @@ async fn create_leaderboard(
     let raw = serde_json::to_string(&lb).map_err(|e| RpcError::Ser(e.to_string()))?;
     let key = format!("leaderboard.{}", leaderboard.id);
     let kv = KeyValueSender::new();
-    kv.set(
-        ctx,
-        &SetRequest {
-            key,
-            value: raw,
-            expires: 0,
-        },
-    )
-    .await?;
+
+    let Ok(res) = kv.get(ctx, &format!("leaderboard.{}", leaderboard.id)).await else {
```

Comment:
Since when has this been syntax?! Cool

Reply:
With this one actually, you may want to use `contains` or at least also check that the `exists` property is true. https://docs.rs/wasmcloud-interface-keyvalue/0.10.0/wasmcloud_interface_keyvalue/struct.GetResponse.html#structfield.exists

I think this could return an Ok(res) even if the value wasn't there